database/mongodb/README.md

```diff
@@ -1,68 +1,98 @@
+ in which the data from the database will be saved are created and are also
+ given the privilege to access and store the data.
 
-At the end of the `%post` section, access is also given to the specific user in the specific group to the database folder in which data is stored.
+At the end of the `%post` section, access is also given to the specific user in
+ the specific group to the database folder in which data is stored.
 
-Then we define a section called `%startscript` which is the section that is executed everytime a container instance is started. In here we find the commands that define a restart of the MongoDB daemon: first a stop of the daemon is executed using `mongod --repair` and then the daemon is started using the `mongod` command. After this, the service can be started with `mongo --host $HOSTNAME:$PORT` in the next line. See that `$HOSTNAME` and `$PORT` correspond to the environment variables we defined and exported at the beginning in the `%environment` section.
+Then we define a section called `%startscript` which is the section that is
+ executed everytime a container instance is started. In here we find the
+ commands that define a restart of the MongoDB daemon: first a stop of the
+ daemon is executed using `mongod --repair` and then the daemon is started using
+ the `mongod` command. After this, the service can be started with
+ `mongo --host $HOSTNAME:$PORT` in the next line. See that `$HOSTNAME` and
+ `$PORT` correspond to the environment variables we defined and exported at the
+ beginning in the `%environment` section.
 
-Another important feature to notice is that you can also refer to these environment variables on runtime with the prefix `$SINGULARITYENV_` , following this example `$SINGULARITYENV_HOSTNAME` and `$SINGULARITYENV_PORT` can be referenced from runtime.
+Another important feature to notice is that you can also refer to these
+ environment variables on runtime with the prefix `$SINGULARITYENV_` , following
+ this example `$SINGULARITYENV_HOSTNAME` and `$SINGULARITYENV_PORT` can be
+ referenced from runtime.
 
 To run the definition file and build our container, we will need to:
 
-```
-$ sudo singularity build mongodb.sif mongodb.def
+```sh
+sudo singularity build mongodb.sif mongodb.def
 ```
 
-This will generate a `mongodb.sif` Singularity image. From there, you will need to bind the `/data/db` folder inside the container to a folder in the host (In this example we will bind it to a new folder in the host called `mongodb/db` ). This folder is where the data from the database such as logs and/or journals are stored. Open a command line and then just run:
+This will generate a `mongodb.sif` Singularity image. From there, you will need
+ to bind the `/data/db` folder inside the container to a folder in the host
+ (In this example we will bind it to a new folder in the host called
+ `mongodb/db` ). This folder is where the data from the database such as logs
```

Comment:
Change line 55 or delete that annotation where it says we're supposed to bind `mongodb/db` folder from host, since on line 61 the binding is now changed to `data/db`